### ISSUE TITLE
NMEA sentences with an included timestamp set the server timestamp value

### ISSUE THREAD

**Issue by wellenvogel** [SignalK/nmea0183-signalk#222]
When receiving NMEA records that include a timestamp this value is used to set the "timestamp" field of the data in signalk. Records that do not include a timestamp use the servers local time.
This way it is nearly impossible to determine the age of a signalK data value as you don't know if the time reference is the gps time or the time of the signalK server (as they easily could differ).
Remark:
NMEA2000 data seems to always use the local server timestamp.

**Comment by tkurki** [comment]
Yes, that doesn't make much sense. We should probably do the opposite: always use sk server time, and add an option to honor the embedded time. But I think that should be on the SK Server side, I think it makes perfect sense that the nmea0183-sk conversion outputs sk timestamps.

**Comment by wellenvogel** [comment]
Should I open a new issue at the server?
But I guess there is still a need for a change at the decoder as - if I understand the code coreectly - the decoder does set up the server local time as timestamp if there is none in the record.

**Comment by tkurki** [comment]
Right. Yes, agreed:
- decoder should not set timestamp if it is not in nmea0183 data
- server should override timestamp unless configured not to do so (separate issue, please)

**Comment by tkurki** [comment]
Changed title to reflect this.
